ocf_data_sampler/torch_datasets/site.py

```diff
@@ -0,0 +1,271 @@
+"""Torch dataset for sites"""
+import logging
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from torch.utils.data import Dataset
+
+from ocf_data_sampler.select.find_contiguous_time_periods import (
+    find_contiguous_t0_periods, find_contiguous_t0_periods_nwp,
+    intersection_of_multiple_dataframes_of_periods,
+)
+from ocf_data_sampler.select.fill_time_periods import fill_time_periods
```

Comment:
Could add some import paths in the select folder init file to tidy up some of these imports